src/components/Dropdown/index.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect, useState } from 'react';
+
+import { ReactComponent as ArrowDropdown } from 'assets/images/icons/arrow-dropdown.svg';
+import { Answer } from 'types/answer';
+
+export const dropdownDataTestIds = {
+  base: 'dropdown__base',
+};
+
+interface DropdownProps {
+  questionId: string;
+  items: Answer[];
+  onValueChanged: (value: Answer) => void;
+}
+const Dropdown = ({ questionId, items, onValueChanged }: DropdownProps): JSX.Element => {
+  const [isOpen, setIsOpen] = useState(false);
+  const [selectedValue, setSelectedValue] = useState(items[0].text);
+
+  const toggleDropdown = () => {
+    setIsOpen((prev) => !prev);
+  };
+
+  const handleOnSelectValue = (value: Answer) => {
+    setSelectedValue(value.text);
+    onValueChanged(value);
+    toggleDropdown();
+  };
+
+  useEffect(() => {
+    setSelectedValue(items[0].text);
+  }, [questionId, items]);
+
+  return (
+    <div className="relative flex flex-col items-center w-full rounded-lg" data-test-id={dropdownDataTestIds.base}>
```

Comment:
Usually, we just use the [`select`](https://www.w3schools.com/tags/tag_select.asp) for the dropdown if there is no design. But building it with React is not that hard, right? 😄

Reply:
@hoangmirs yeah, I'm trying to create a custom component on my own and the result is not bad 🚀